src/index.html

```diff
@@ -11,8 +11,63 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <!-- fonts -->
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <!-- crossorigin="attr-req-value" -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
+    <!-- fonts -->
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="img-computer"
+      />
+
+      <div class="card__bottom">
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <div class="card__product-code">Product code: 195434</div>
+
+        <div class="stars card__stars">
+          <div class="stars__block">
```

Comment:
```suggestion
          <div class="stars stars--4">
```
Consider using class from the previous task